includes/Plugin.php

```diff
@@ -157,6 +157,7 @@ function() use ( $options, $transients, $assets ) {
 				( new Core\Admin\Dashboard( $this->context, $assets, $modules ) )->register();
 				( new Core\Notifications\Notifications( $this->context, $options ) )->register();
 				( new Core\Util\Debug_Data( $this->context, $options, $user_options, $authentication, $modules ) )->register();
+				new Core\Admin\Standalone( $this->context );
```

Comment:
See other comment about how to add hooks in Site Kit, let's call the `register` method on the instance (like in the lines above).